source/runner/Executor.ts

```diff
@@ -70,7 +70,7 @@ export interface ExecutorOptions {
 const isTests = typeof jest === "object"
 
 interface ExitCodeContainer {
-  exitCode: number
+  exitCode?: number
```

Comment:
because TS was updated it started to complain about 

<img width="1611" alt="Screenshot 2021-03-22 at 17 51 36" src="https://user-images.githubusercontent.com/2836281/112018468-40b50780-8b37-11eb-9b0a-7215ec3e4877.png">

So had to fix this type to fix it